src/core/sys/posix/aio.d

```diff
@@ -34,6 +34,66 @@ version (CRuntime_Glibc)
             off_t aio_offset;
             ubyte[32] __glibc_reserved;
         }
+
+    }
+    else version (FreeBSD)
+    {
+        struct __aiocb_private {
+            long status;
+            long error;
+            void *kernelinfo;
```

Comment:
Should probably use D style type here, and in other places (`void* foo`, not `void *foo`).

Reply:
Q:Would it make sense to to extend checkwhitespace.d to also check for this ?

Reply:
We should have a much more comprehensive lint used for phobos, so we should probably replace checkwhitespace with that and spend effort on something that may be replaced in the future.